cmake/ExperimentalEmscripten.cmake

```diff
@@ -0,0 +1,112 @@
+#
```

Comment:
I'd suggest putting this file into `cmake/Platform/Emscripten-STLab.cmake`. The `Platform` subdirectory for toolchains is used both in CMake and in Emscripten packages. Dash is used for word separators in CMake toolchain files. The STLab distinguishes this toolchain file from `Emscripten.cmake`.

Reply:
Sounds great. Can do.